dist/DeadlineCloudSubmitter.jsx

```
@@ -2412,13 +2412,15 @@ function buildUI(thisObj) {
 
     var submitButton = controlsGroup.add("button", undefined, "Submit");
     submitButton.onClick = function() {
-        const multiFrameRendering = mfrCheckBox.value ? "ON" : "OFF";
-        var maxCpuUsagePercentage = undefined;
-        if (mfrCheckBox.value) {
-            maxCpuUsagePercentage = parseInt(maxCpuUsagePercentageTextBox.text)
+        if (getPythonExecutable()) {
+            const multiFrameRendering = mfrCheckBox.value ? "ON" : "OFF";
+            var maxCpuUsagePercentage = undefined;
+            if (mfrCheckBox.value) {
+                maxCpuUsagePercentage = parseInt(maxCpuUsagePercentageTextBox.text)
+            }
+            SubmitSelection(list.selection, parseInt(framesPerTaskTextBox.text), multiFrameRendering, maxCpuUsagePercentage);
+            list.selection = null;
         }
```

If we would like to do this check, I think we should throw the error in a else block to notify customers that their python executable is not set as expected and pointing them to any possible reference to unblock the issue.

---

The error message happens in the getPythonExecutable function call

---

The getPythonExectuable function has error handling built into that because it's used in other places.